app/views/tracks/show/joined.html.haml

```diff
@@ -0,0 +1,60 @@
+#page-track-dashboard
+  %nav.track-nav
+    .lg-container.container
+      = link_to "<-", tracks_path, class: "back"
+      = image_tag(@track.icon_url, class: 'c-track-icon')
+      .title= @track.title
+      .tabs
+        = link_to "#", class: 'selected' do
+          Overview
+
+        = link_to track_concepts_path(@track) do
+          Concepts
+
+        = link_to "#" do
+          Exercises
+
+  %header.track-header
+    .lg-container.container
+      = image_tag(@track.icon_url, class: 'c-track-icon')
+      .info
+        %h1= @track.title
+        .joined Joined on #{format_date(@user_track.created_at)}
+      .progress-bar
+
+  %article
+    .lg-container.container
+      .lhs
+      .rhs
+        %section.contributors
+          %header
+            %h2 Contributors
+            .count 123
+          .showcase
+            = link_to "#" do
+              .img{ style: "background-image:url(https://avatars3.githubusercontent.com/u/135246?s=460&u=733b8575c8f0dbb5a52840c3efb2480b166c897a&v=4)" }
+              .name Erik Schierboom
+              .reputation
+                = reputation_icon(color: "#8480A0")
+                123
+            = link_to "#" do
+              .img{ style: "background-image:url(https://avatars2.githubusercontent.com/u/17816791?s=460&u=75b8876b2ec4d897e946888493a7fc7b7d97ec55&v=4)" }
+              .name Rob Keim
+              .reputation
+                = reputation_icon(color: "#8480A0")
+                456
+          .faces
+            = link_to "", "#", class: 'face', style: "background-image:url(https://avatars3.githubusercontent.com/u/2716614?s=460&u=7403dc7f80ccfd983705a1637fac1b12d2dcbad2&v=4)"
+            = link_to "", "#", class: 'face', style: "background-image:url(https://avatars1.githubusercontent.com/u/20866761?s=400&u=9444c34f830fe94f2b68995646ed405a65911a91&v=4)"
+            = link_to "", "#", class: 'face', style: "background-image:url(https://avatars3.githubusercontent.com/u/6624571?s=400&u=ca3801c9d18d8aa124738c8e8db470cbe5fd5814&v=4)"
+            = link_to "", "#", class: 'face', style: "background-image:url(https://avatars2.githubusercontent.com/u/7852553?s=400&u=6227e3aeff0e649cce5483fca11deb3cc6481319&v=4)"
+            = link_to "", "#", class: 'face', style: "background-image:url(https://avatars2.githubusercontent.com/u/281700?s=400&v=4)"
+            = link_to "", "#", class: 'face', style: "background-image:url(https://avatars0.githubusercontent.com/u/6251883?s=400&u=1111c5251e1c1bbf6b45790b6ffa4a919c502bd6&v=4)"
+            = link_to "", "#", class: 'face', style: "background-image:url(https://avatars2.githubusercontent.com/u/1964376?s=400&u=591370994dd91ec3d5f2a345252a653406858be0&v=4)"
+            = link_to "", "#", class: 'face', style: "background-image:url(https://avatars2.githubusercontent.com/u/365425?s=400&v=4)"
+            = link_to "See all", "#", class: "all"
+          .cta
+            %h3 We 💙 our contributors
```

Comment:
Suggest to wrap that `💙` inside e.g. a `span` or `i`, and add `aria-label`.

Reply:
👍